### ISSUE TITLE
Gold loot problem when in group (BB #925)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#925]
This issue was migrated from bitbucket.
**Original Reporter:** Dikkedeur
**Original Date:** 28.05.2015 08:30:22 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925

<hr>

Revision: https://bitbucket.org/oregon/oregoncore/commits/3407fe129414c4cbf4b8c986c5b171b3f2946212
customcode: Some anticheat patch nothing related to looting.
database: Oregondatabase.
OS: Ubuntu server

problem: When you are in a group and loot gold it sometimes doesnt add the gold to the player(s)
how to reproduce: Go ingame with fresh chars, i made 2 human mages in human territory, then group up ( /invite ) then levelup a bit, at some point when some npc drops gold ( in this case copper since we start with fresh toons ) it might be random if the copper is added, most of the time its 30% to 50% fail.
Ofcourse when the loot is 1 copper it can be divided, since the lowest amount to loot is 1 copper and it can be divided between 2 characters, but when you loog 4 copper, then 2 copper should go to player A and Player B also gets 2 copper.
Well now its totally random and it might be taking a few strikes before you encounter this bug.

how it should work: When you are in a party and loog gold or copper it should be divided equal between the players.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Dikkedeur
**Original Date:** 28.05.2015 08:31:14 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18421828

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 28.05.2015 18:36:03 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18436263

<hr>

To do : confirm it's not visual, if it is..look at loot msg notify

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Dikkedeur
**Original Date:** 28.05.2015 21:16:37 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18439298

<hr>

no its not visual, i had 0 copper it was a new char, and when i looted 4 copper i should have had 2 copper ( 2 for me and 2 for the other ) there was also in chat message not the message like it should.
your share of the loot 2 copper.
there was nothing.

so its not only visual in chat, its not being processed at all.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 29.05.2015 03:04:17 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18443366

<hr>

All right, confirmed and I've worked out what it is. Working on it now!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 29.05.2015 03:26:58 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18443514

<hr>

[Core/Loot] Prevent sharing money loot when opening items, pickpocketing or looting corpses.
            Correctly send loot error incase of can't loot creature because of distance.
            Send NotifyMoneyRemoved before ModMoney.
            Uncomment "Interrupt Spell casting on loot"
            Correctly determine whether a player can loot money or not in a group.

```
        Fixes #925. Thanks for the detailed report! We appreciate it and
        thank our users who are thoughtful :)
```

→ <<cset 71931c45d710>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Dikkedeur
**Original Date:** 29.05.2015 05:54:17 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/925/#comment-18444821

<hr>

no problem, i think that indeed bugreports should be for the sake of the devs in a special format.
so like.

Rev:
DB:
Customcode:
OS:
Problem:
How to reproduce:
How it should work:

i have no idea if its possible on bitbucket if you can set templates for bugreports...
